css/index.css

```diff
@@ -97,71 +95,74 @@
      text-align: center;
      padding: 16px 0px;
        }
-         }
-     
          .menu li + li {
         margin-top: 12px;
         }
-   
-     .about-section {
-      padding: 50px;
-      text-align: center; 
-      color: white;
-       }
-        <!--APPLYING SECTION QUERIES-->
+
+        input [type=checkbox]:checked ~ .menu { 
+          display: inline-block;
+        }     
+      }
+    
+        /*APPLYING SECTION QUERIES*/
+
       .about-section {
        width: 100%;
-       height: 50px;
+       height: 100vh;
+       padding:0 150px;
        display: flex;
        align-items: center;
        position: fixed;
-       top: 20px;
+       top: 0;
        opacity: 0;
        transition: 1s;
        
        }
-   
-      .hero-heading{
-       color: #fff;
-       font-size: 70px;
-       text-transform: capitalize;
-       font-weight: 60px;
-       top:90px;
-       text-shadow:2px 1px yellow;			
-        }
-   
+       .hero-heading{
+        color: #fff;
+        font-size: 120px;
+        text-transform: capitalize;
+        font-weight: 300;
+        text-shadow:0 0 10px gold, 0 0 20px firebrick, 0 0 40px pink, 0 0 80px red ;			
+      }   
+      
        .about-img{
        position: absolute;
-       top: 90px;
+       top: 75px;
        right: 0;
-       height: 120vh;
-       width: 30%;
+       height: 100vh;
+       width: 40%;
        object-fit: cover;
        opacity: 0.2;
         }
+
       .about-info{
        color: #fff;
        opacity: 0.6;
-       line-height: 40px;
+       line-height: 50px;
        font-size: 30px;
           } 
-    About.active, 
-    project.active, 
-    experience.active, 
-    connect.active {
+    .about-section.active, 
+    .experience-section.active,
+    #skillSec.active,
+    #project.active,
+    .message-section.active,
+    .connect-section.active{
```

Comment:
I think these are all unique, so you might make all the sections IDs instead of classes.

If these are unique, I'm not sure what the purpose of the `.active` is.